plugins/package-managers/python/src/main/kotlin/Pip.kt

```diff
@@ -130,4 +130,12 @@ class Pip(
             }
         }.getOrThrow()
     }
+
+    private fun detectPythonVersion(workingDir: File): String? {
+        // While there seems to be no formal specification, a `.python-version` file seems to be supposed to simply
+        // conatin the plan version, see e.g. https://github.com/pyenv/pyenv/blob/21c2a3d/test/version.bats#L28.
```

Comment:
typo: 
- splan -> plain
- conatin -> contain

maybe add a "just" before contain?

Reply:
> maybe add a "just" before contain?

As it should be either "just" or "simply", IMO, I've swapped out "simply" for "just".